go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/omeryahud/ssp-operator
+
+go 1.13
+
+require (
+	github.com/operator-framework/operator-sdk v0.18.2
+	github.com/spf13/pflag v1.0.5
+	k8s.io/api v0.18.2
+	k8s.io/apimachinery v0.18.2
+	k8s.io/client-go v12.0.0+incompatible
+	sigs.k8s.io/controller-runtime v0.6.0
+)
+
+replace (
```

Comment:
Why do these packages need to be in the `replace` block and not only in `require`?

Reply:
I'm not sure here, this was generated by `operator-sdk` binary

Reply:
Ok, I was just curious. If it was generated, we can assume there was a reason for it.

Reply:
I verified it on a example operator also, this is default generated for me